lib/tasks/send_survey_notifications.rake

```diff
@@ -5,17 +5,24 @@ namespace :survey_notifications do
     BATCH_SIZE = 10
     accepted_submissions = EfileSubmission.joins(:efile_submission_transitions)
                                           .where("efile_submission_transitions.to_state = 'accepted'")
+                                          .where("efile_submissions.data_source_type = 'StateFileAzIntake' OR efile_submissions.data_source_type = 'StateFileNyIntake'")
                                           .where.not("message_tracker #> '{messages.state_file.survey_notification}' IS NOT NULL")
 
     accepted_submissions.each_slice(BATCH_SIZE) do |batch|
       batch.each do |submission|
-        puts "Sending survey notification to #{submission.id}"
-        StateFile::MessagingService.new(
-          intake: submission.data_source,
-          submission: submission,
-          message: StateFile::AutomatedMessage::SurveyNotification,
-          body_args: { survey_link: survey_link(submission.data_source) }
-        ).send_message
+        intake = submission.data_source
+        if intake.nil?
```

Comment:
you can also do `submission.is_for_federal_filing?` (checks if its a submission for CTC) or `submission.is_for_state_filing?` (checks for state-file)